lib/validators/specValidator.js

```diff
@@ -603,22 +596,35 @@ class SpecValidator {
       }
       let location = parameter.in;
       if (location === 'path' || location === 'query') {
+        if (location === 'path' && parameterValue && typeof parameterValue.valueOf() === "string") {
+          if (options.pathTemplate.charAt(options.pathTemplate.indexOf(`${parameter.name}`) - 2) === '/' && parameterValue.startsWith('/')) {
+            let msg = `In operation "${operation.operationId}", example for parameter "${parameter.name}": "${parameterValue}" starts with a forward slash ` +
+              `and the path template: "${options.pathTemplate}" contains a forward slash before the parameter starts. This will cause double forward slashes ` +
+              ` in the request url. Thus making it incorrect. Please rectify the example.`;
+            let e = self.constructErrorObject(ErrorCodes.DoubleForwardSlashesInUrl, msg);
+            result.validationResult.errors.push(e);
+            foundIssues = true;
```

Comment:
I am still iffy about this. Not sure if throwing an error for this scenario is necessary.

Reply:
I think throwing would give more visibility than swallowing...

Reply:
If we throw we stop any other validation, right? If we log an errors, we report it as any other? If so, I'd go with error.

Reply:
I think my sentence in the comment confused you guys. Sorry about that
> I am still iffy about this. Not sure if throwing an error for this scenario is necessary.

Yes we are just logging the error like we do for other scenarios. We are not throwing and halting further validation. 
What I actually meant to ask is 
> "Does it actually make sense to catch these kind of issues? I mean in the real world customers could provide forward slashes in their parameter values when they use an sdk or the portal or REST API directly. Shouldn't this be handled by the service. (they should ignore duplicate forward slashes or put a regex validation constraint in swagger spec that the value should not start with forward slash". Making this as a brute force validation in the model validator is too strict.

What do you think?

Reply:
yeah, I thought about that while reviewing too, I think we could just swallow honestly for now and fix it "silently", I think having a warning instead of an error could be a good thing, in case we decide to use this value for something else later and the impact is different - we would at least know it's flagged.